### DISCUSSION TITLE
Why does Database Cluster require a VPC when configured to use the Data API

### DISCUSSION THREAD

**Discussion by raymondKelly** [aws/aws-cdk#30616] [Q&A]
When using the Database Cluster 

- https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-rds/lib/cluster.ts
- https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds-readme.html#starting-a-clustered-database

Why is a VPC required in CDK when using the RDS Data API? 

The documentation also declares that the VPC is optional https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds-readme.html#default-vpc
but If I create a database cluster without a VPC such as the Data API example mentions https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds-readme.html#data-api
then I get the following error

```
Error: Provide either vpc or instanceProps.vpc, but not both
    at new DatabaseClusterNew (/Users/kellyr/fineos-bitbucket-repos/prisma-rds-data-api/node_modules/aws-cdk-lib/aws-rds/lib/cluster.js:1:4224)
    at new DatabaseCluster (/Users/kellyr/fineos-bitbucket-repos/prisma-rds-data-api/node_modules/aws-cdk-lib/aws-rds/lib/cluster.js:5:4506)
```

**Comment by zfoster** [comment]
I have the same issue, seems to occur regardless of whether dataApi is enabled. Are we unable to use the default VPC?

**Comment by Gin-Quin** [comment]
I have this error as well. This is weird, because the error message makes no sense: "Provide either vpc or instanceProps.vpc, but not both". I didn't provide "both", I provided none of them (wanting to use the default vpc).

Anyone could solve this?
